### ISSUE TITLE
does not configure under 5.26

### ISSUE THREAD

**Issue by jasonblewis** [cho45/Test-Time#5]
when trying to install with cpanminus, it fails on the configure step

```

Written by John Gilmore and Jay Fenlason.
You have /usr/bin/unzip
Searching Test::Time () on cpanmetadb ...
--> Working on Test::Time
Fetching http://www.cpan.org/authors/id/S/SA/SATOH/Test-Time-0.04.tar.gz
-> OK
Unpacking Test-Time-0.04.tar.gz
Entering Test-Time-0.04
Checking configure dependencies from META.yml
Checking if you have ExtUtils::MakeMaker 6.58 ... Yes (7.28)
Configuring Test-Time-0.04
Running Makefile.PL
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/site_perl/5.26.0/x86_64-linux /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/site_perl/5.26.0 /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/5.26.0/x86_64-linux /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/5.26.0) at Makefile.PL line 2.
BEGIN failed--compilation aborted at Makefile.PL line 2.
-> N/A
-> FAIL Configure failed for Test-Time-0.04. See /home/jason/.cpanm/work/1497084349.738/build.log for details.
```

**Comment by jasonblewis** [comment]
```
jason@vm:~/.cpanm/work/1497084515.803/Test-Time-0.04$ perl Makefile.PL 
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/site_perl/5.26.0/x86_64-linux /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/site_perl/5.26.0 /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/5.26.0/x86_64-linux /home/jason/perl5/perlbrew/perls/perl-5.26.0/lib/5.26.0) at Makefile.PL line 2.
BEGIN failed--compilation aborted at Makefile.PL line 2.
```

**Comment by kentfredric** [comment]
Fixed in 0.05
